snuba/query/snql/anonymize_visitor.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date, datetime
+
+from snuba.query.expressions import Literal, StringifyVisitor
+
+
+class AnonymizeAndStringifySnQLVisitor(StringifyVisitor):
```

Comment:
You do not need to rebuild this.
https://github.com/getsentry/snuba/blob/master/snuba/clickhouse/formatter/expression.py#L205

Now it relies on ClickhouseExpressionFormatterBase, but that is not a blocker for you. 
In every case you can turn ClickhouseExpressionFormatterBase into `ExpressionFormatterBase` as there is really not anything clickhouse specific in that class.